front/src/App.css

```
@@ -20,8 +26,9 @@ body {
     width: 100%;
     display: flex;
     flex-direction: column;
-    flex-wrap: wrap;
+    flex-wrap: scroll;
```

```suggestion
    flex-wrap: wrap;
```

Un reste de mes fake news :)